### PR TITLE
CNV-23499: 4.10+ HPP CSI only (4.12 RN update)

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s):4.12

Issue: [CNV-23499](https://issues.redhat.com/browse/CNV-23499)

Link to docs preview: [https://57355--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-12-release-notes.html#virt-4-12-removed](https://57355--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-12-release-notes.html#virt-4-12-removed)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by aspauldi** [comment]
@aglitke, Can you please take a look at this update to the 4.12 release notes? 

This change for CNV-23499 reflects only the moving of information about newly installed clusters deploying only HPP CSI from the Deprecated features to the Removed feature section of the notes and some slight modification of the note itself. An additional PR wll cover non-release-note content changes. Thanks.

**Comment by aspauldi** [comment]
@[kgoldbla@redhat.com](mailto:kgoldbla@redhat.com), Can you please take a look at this update to the 4.12 release notes?

File: virt/virt-4-12-release-notes.adoc [hunk 1]
```diff
@@ -134,20 +134,20 @@ The SVVP Certification applies to:
...
+// Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
 
 
 [id="virt-4-12-removed"]
 === Removed features
 
 Removed features are not supported in the current release.
 
+* Support for the legacy HPP custom resource, and the associated storage class, has been removed. Beginning in {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. 
+
```
Comment (kgoldbla) [inline virt/virt-4-12-release-notes.adoc:150]: Beginning in {VirtProductName} {VirtVersion} - > Not sure whether you are stating when this change was made. If so it was made in OpenShift Virtualization 4.10. If you are referring to how thing works in the current version then change 'Beginning in'  to 'In'

File: virt/virt-4-12-release-notes.adoc [hunk 2]
```diff
@@ -140,14 +140,17 @@ The SVVP Certification applies to:
...
+ 
 
 
 [id="virt-4-12-removed"]
 === Removed features
 
 Removed features are not supported in the current release.
 
+// CNV-23499_412RN
+* Support for the legacy HPP custom resource, and the associated storage class, has been removed. In {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage.
```
Comment (kgoldbla) [inline virt/virt-4-12-release-notes.adoc:152]: Sorry for the delay. I had to clarify this with the storage development team stakeholders. Please make the following change:
Support for the legacy HPP custom resource, and the associated storage class, has been removed. -> Support for the legacy HPP custom resource, and the associated storage class, has been removed for all new deployments. 

In {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. -> [Add this] Only in the case of an upgrade will a previously installed legacy HPP custom resource still be supported.

**Comment by aspauldi** [comment]
@kgoldbla , Thanks for the comments. Can you take another look?

**Review by kgoldbla — COMMENTED** [review]
lgtm

File: virt/virt-4-12-release-notes.adoc (hunk 1 above)
Comment (eohartman) [reply virt/virt-4-12-release-notes.adoc:153]: I feel like the "Only in the case of..." sentence is a bit wordy and awkward. 
Maybe change it to "Legacy HPP custom resource is supported only if it had been installed on a previous version of {VirtProductName}."

**Review by eohartman — CHANGES_REQUESTED** [review]

File: virt/virt-4-12-release-notes.adoc (hunk 1 above)
Comment (kgoldbla) [reply virt/virt-4-12-release-notes.adoc:153]: lgtm
